mongodb/README.md

```diff
@@ -0,0 +1,249 @@
+default    pvc-3ee3c186-a7a7-4035-893f-e66607e73c2e  1.0 GiB  worker1 (online)  worker1      0/0       6 minutes ago
+default    pvc-fabbdce2-9b42-4e4f-a719-ee18e56c1195  1.0 GiB  worker3 (online)  worker3      0/0       8 minutes ago
+```
+
+Get the list of `PVCs` to map them out to the Ondat volumes displayed above:
+```
+kubectl get pvc
+```
+Result:
+```
+NAME                    STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+data-volume-mongodb-0   Bound    pvc-85179a6f-44d2-49c6-8b11-d96bd91009b2   1Gi        RWO            ondat-replicated   11m
+data-volume-mongodb-1   Bound    pvc-f70ac39b-b74f-4b2e-b47c-9b49e14ae25b   1Gi        RWO            ondat-replicated   10m
+data-volume-mongodb-2   Bound    pvc-3ee3c186-a7a7-4035-893f-e66607e73c2e   1Gi        RWO            ondat-replicated   9m46s
+logs-volume-mongodb-0   Bound    pvc-fabbdce2-9b42-4e4f-a719-ee18e56c1195   1Gi        RWO            ondat-replicated   11m
+logs-volume-mongodb-1   Bound    pvc-afbb7b28-a8f3-4238-966f-76b2a3ebcf9c   1Gi        RWO            ondat-replicated   10m
+logs-volume-mongodb-2   Bound    pvc-723f1945-5c99-441d-a8a3-f43cc6d0c668   1Gi        RWO            ondat-replicated   9m46s
+```
+
+In the example above, you can see that `mongodb-0` data volume PVC ID is ending with `9b2`. If you check this ID in the Ondat volumes list, you can also find this volume ending with `9b2`. This information allows you to identify the node where the data volume is attached to. In this example, the node is `worker3` (use the column `ATTACHED ON`).
+
+Let's check that there's no trace of the string `rose` in the encrypted data volume of `mongodb-0`. You can now choose any other MongoDB `Pod` if you fancy! The data is replicated to the other two nodes. Just make sure you target the right Kubernetes worker node (ie. where the Ondat volume is attached to).
+
+But first, verify that the string rose is present within the database binary file located under /data by executing the following command:
+```
+kubectl exec -it mongodb-0 -- grep -r rose /data
+```
+Result:
+```
+Binary file /data/journal/WiredTigerLog.0000000002 matches
+Binary file /data/collection-16-6361445414969153344.wt matches
+Binary file /data/collection-6--5148538780104397675.wt matches
+Binary file /data/collection-10--5148538780104397675.wt matches
+```
+
+You can notice that there is a match with multiple files.
+
+Now, let's verify if the data is encrypted when accessing the raw volume. This is simulating an attacker who leveraged privilege escalation to get access to the storage device. The string `rose` should not be visible when the attacker tries to access the encrypted data. Let's verify this.
+
+Connect to the node where the Ondat volume is attached to (column `ATTACHED ON` above). In our case, this is node `worker3`. Run the following command:
+```
+ssh root@worker3 strings /var/lib/storageos/data/dev1/vol.*.blob | grep rose
```

Comment:
```suggestion
ssh root@worker3 grep 'rose' /var/lib/storageos/data/dev1/vol.*.blob
```

This should be enough, `strings` is quite slow and prone to vulnerabilities if somehow somebody could add invalid data into the volume.

as evidenced:
```bash
root@ip-192-168-67-97:/var/lib/storageos/data/dev1# time strings /var/lib/storageos/data/dev1/*.blob  ö grep rose

real    0m0.529s
user    0m0.460s
sys     0m0.039s
root@ip-192-168-67-97:/var/lib/storageos/data/dev1# time grep rose /var/lib/storageos/data/dev1/*.blob

real    0m0.021s
user    0m0.016s
sys     0m0.005s
```

Reply:
I see! It does mean the user will need to install `strings` on their node if they don't already have it, so perhaps a note to that effect is fine.

Reply:
Alternatively, we could change the blog post!